File: backend/Chess.DataAccess/Entities/Side.cs
using Chess.DataAccess.Helpers;
using System;
using System.Collections.Generic;
using System.Text;

namespace Chess.DataAccess.Entities
{
    public class Side : Entity
    {
        public virtual Game Game { get; set; }
        public int? GameId { get; set; }
        public virtual User Player { get; set; }
        public int? PlayerId { get; set; }
        public Color Color { get; set; }
        public int Points { get; set; }
        public bool IsDraw { get; set; }
        public bool IsResign { get; set; }
    }
}

File: frontend/src/app/core/models/chess/side.ts
import { Game } from "./game";
import { User } from "../user";
import { GameSide } from "./gameSettings";

export class Side {
    id?: number;
    game?: Game;
    gameId?: number;
    points?: number;
    isDraw?: boolean;
    isResign?: boolean;
    constructor(
        public color: GameSide,
        public player?: User
    ) {}
}